manifest.json

```diff
@@ -9,8 +9,10 @@
 		"128": "icon.png"
 	},
 	"permissions": [
-		"https://github.com/*/*"
+		"https://github.com/*/*",
+		"storage"
 	],
+	"options_page": "options.html",
```

Comment:
Use the new Chrome options UI: https://github.com/sindresorhus/notifier-for-github-chrome/blob/994bae39590596754fbc9c482663342e51999791/extension/manifest.json#L27-L30

You could base it on https://github.com/sindresorhus/notifier-for-github-chrome/blob/994bae39590596754fbc9c482663342e51999791/extension/options.html

No need for the Bootstrap stuff.